README.md

```diff
@@ -13,111 +13,84 @@ This plugin intends to support linting of ES2015+ (ES6+) import/export syntax, a
+❌ Deprecated.
 
 ### Helpful warnings
 
-
-* Report any invalid exports, i.e. re-export of the same name ([`export`])
-* Report use of exported name as identifier of default export ([`no-named-as-default`])
-* Report use of exported name as property of default export ([`no-named-as-default-member`])
-* Report imported names marked with `@deprecated` documentation tag ([`no-deprecated`])
-* Forbid the use of extraneous packages ([`no-extraneous-dependencies`])
-* Forbid the use of mutable exports with `var` or `let`. ([`no-mutable-exports`])
-* Report modules without exports, or exports without matching import in another module ([`no-unused-modules`])
-* Prevent empty named import blocks ([`no-empty-named-blocks`])
-
-[`export`]: ./docs/rules/export.md
-[`no-named-as-default`]: ./docs/rules/no-named-as-default.md
-[`no-named-as-default-member`]: ./docs/rules/no-named-as-default-member.md
-[`no-deprecated`]: ./docs/rules/no-deprecated.md
-[`no-extraneous-dependencies`]: ./docs/rules/no-extraneous-dependencies.md
-[`no-mutable-exports`]: ./docs/rules/no-mutable-exports.md
-[`no-unused-modules`]: ./docs/rules/no-unused-modules.md
-[`no-empty-named-blocks`]: ./docs/rules/no-empty-named-blocks.md
+| Name                                                                   | Description                                                                           | 💼                                  | 🔧 | 💡 | ❌  |
+| :--------------------------------------------------------------------- | :------------------------------------------------------------------------------------ | :---------------------------------- | :- | :- | :- |
+| [export](docs/rules/export.md)                                         | Forbid any invalid exports, i.e. re-export of the same name.                          | ❗ ☑️                                |    |    |    |
+| [no-deprecated](docs/rules/no-deprecated.md)                           | Forbid imported names marked with `@deprecated` documentation tag.                    |                                     |    |    |    |
+| [no-empty-named-blocks](docs/rules/no-empty-named-blocks.md)           | Forbid empty named import blocks.                                                     |                                     | 🔧 | 💡 |    |
+| [no-extraneous-dependencies](docs/rules/no-extraneous-dependencies.md) | Forbid the use of extraneous packages.                                                |                                     |    |    |    |
+| [no-mutable-exports](docs/rules/no-mutable-exports.md)                 | Forbid the use of mutable exports with `var` or `let`.                                |                                     |    |    |    |
+| [no-named-as-default](docs/rules/no-named-as-default.md)               | Forbid use of exported name as identifier of default export.                          | ☑️<sup>⚠️</sup> <br>🚸<sup>⚠️</sup> |    |    |    |
```

Comment:
it seems like the superscripted emoji is still rendering on a new line, and isn't staying attached to the base emoji?

Reply:
Ugh. I fixed some other issues with this but looks like I'm still struggling to get the emoji superscript to stay attached to its base character. Tracking this with: https://github.com/bmish/eslint-doc-generator/issues/204

Reply:
I tried everything to fix the emoji superscripts but they turned out to be too fragile. This led me to remove the superscripts entirely and switch to what I believe is a significantly better information architecture with a separate rules list column per severity and many other improvements:
* https://github.com/bmish/eslint-doc-generator/pull/205